src/sum-of-odd.js

```diff
@@ -25,7 +25,7 @@
  * @return {number}
  */
 function rowSumOddNumbers(row) {
-  // write code here
+  return (row ** 3) <= 0 ? 0 : (row ** 3);
```

Comment:
try to use `Math.pow()`
no need for this check `(row ** 3)`
```suggestion
  return row <= 0 ? 0 : (row ** 3);
```